content/docs/hello-world.md

```diff
@@ -1,50 +1,50 @@
 ---
 id: hello-world
-title: Hello World
+title: Bonjour monde
 permalink: docs/hello-world.html
 prev: cdn-links.html
 next: introducing-jsx.html
 ---
 
-The smallest React example looks like this:
+Le plus petit exemple de React ressemble à ceci:
 
 ```js
 ReactDOM.render(
-  <h1>Hello, world!</h1>,
+  <h1>Bonjour, monde !</h1>,
   document.getElementById('root')
 );
 ```
 
-It displays a heading saying "Hello, world!" on the page.
+Il affiche un titre de section mentionnant « Bonjour, monde ! » sur la page.
 
 [](codepen://hello-world)
 
-Click the link above to open an online editor. Feel free to make some changes, and see how they affect the output. Most pages in this guide will have editable examples like this one.
+Cliquez sur le lien ci-dessus pour ouvrir un éditeur en ligne. Vous êtes libres de faire quelques changements et de voir comment ils affectent l'affichage. La plupart des pages de ce guide auront des exemples modifiables comme celui-ci.
 
 
-## How to Read This Guide {#how-to-read-this-guide}
+## Comment lire ce guide {#how-to-read-this-guide}
 
-In this guide, we will examine the building blocks of React apps: elements and components. Once you master them, you can create complex apps from small reusable pieces.
+Dans ce guide, nous examinerons les éléments constituants les applications React: éléments et composants. Une fois maîtrisés, vous pourrez créer des applications complexes à partir de petites pièces réutilisables.
```

Comment:
```suggestion
Dans ce guide, nous examinerons les blocs qui constituent une application React : les éléments et les composants. Une fois que vous les aurez maîtrisés, vous pourrez créer des applications complexes à partir de petites pièces réutilisables.
```

- On va éviter « éléments constituants », pour deux raisons. Primo, « éléments » est très connoté en React, d'ailleurs tu t'en sers juste après dans ce sens-là. Secundo, on essaie d'éviter les participes présents comme adjectifs.
- On parle bien de maîtriser ces blocs, donc oui, un « s », mais justement on gagne à expliciter la tournure, vu que la subordonnée ne s'applique pas à eux.

Reply:
@setheal quid de cette suggestion ? tu ne l'as pas validée ? (en passant par GitHub, onglet *Files changed*, bouton *Add suggestion to batch*, par exemple)

Reply:
J'ai un joli "Outdated suggestions cannot be applied", je vais les ajouter à la main :)